pymove/_version.py

```diff
@@ -1 +1 @@
-__version__ = '1.5.0'
+__version__ = '2.0.0'
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6041015)